docs/content/connectors/oceanbase-cdc(ZH).md

```diff
@@ -0,0 +1,238 @@
+# Flink OceanBase CDC 连接器
+
+OceanBase CDC 连接器允许从 OceanBase 读取快照数据和增量数据。本文介绍了如何设置 OceanBase CDC 连接器以对 OceanBase 进行 SQL 查询。
+
+## 依赖
+
+要设置 OceanBase CDC 连接器，您必须提供相关的依赖信息。以下依赖信息适用于使用自动构建工具（如 Maven 或 SBT）构建的项目和使用 SQL JAR 包捆绑的 SQL 客户端。
+
+```xml
+<dependency>
+  <groupId>com.ververica</groupId>
+  <artifactId>flink-connector-oceanbase-cdc</artifactId>
+  <!-- The dependency is available only for stable releases, SNAPSHOT dependency need build by yourself. -->
+  <version>2.2.1</version>
+</dependency>
+```
+
+## 下载 SQL 客户端 JAR 包
+
+点击 [Maven 中央仓库](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-oceanbase-cdc/2.2.1/flink-sql-connector-oceanbase-cdc-2.2.1.jar) 下载 JAR 包至 `<FLINK_HOME>/lib/`.
+
+> **说明：**
+>
+> 下载链接仅适用于稳定发行版本。
+
+`flink-sql-connector-oceanbase-cdc-XXX-SNAPSHOT` 快照版本与开发分支的版本对应。要使用快照版本，您必须自行下载并编译源代码。推荐使用稳定发行版本，例如 `flink-sql-connector-oceanbase-cdc-2.2.1.jar`。您可以在 Maven 中央仓库中找到使用稳定发行版本。
+
+### 配置 OceanBase 数据库和 oblogproxy
+
+1. 按照 [部署文档配置](https://open.oceanbase.com/docs/community/oceanbase-database/V3.1.1/deploy-the-distributed-oceanbase-cluster) OceanBase 集群。
+2. 在 sys 租户中，为 oblogproxy 创建用户。更多信息，参考 [用户管理文档](https://open.oceanbase.com/docs/community/oceanbase-database/V3.1.1/create-user-3)。
+
+    ```bash
+    mysql -h${host} -P${port} -uroot
+    mysql> SHOW TENANT;
+    mysql> CREATE USER ${sys_username} IDENTIFIED BY '${sys_password}';
+    mysql> GRANT ALL PRIVILEGES ON *.* TO ${sys_username} WITH GRANT OPTION;
+    ```
+
+3. 为待监控有租户创建一个租户。此租户用来读取快照数据和变化事件数据。
+4. 使用以下命令，获取 `rootservice_list` 的值。
+
+    ```bash
+    mysql> SHOW PARAMETERS LIKE 'rootservice_list';
+    ```
+
+5. 按照 [oblogproxy 文档](todo) 配置 oblogproxy。
+
+## 创建 OceanBase CDC 表
+
+使用以下命令，创建 OceanBase CDC 表：
+
+```sql
+-- 每 3000 毫秒检查一次                   
+Flink SQL> SET 'execution.checkpointing.interval' = '3s';
+
+-- 在 Flink SQL 中创建 OceanBase 表 `orders`
+Flink SQL> CREATE TABLE orders (
+    order_id     INT,
+    order_date   TIMESTAMP(0),
+    customer_name STRING,
+    price        DECIMAL(10, 5),
+    product_id   INT,
+    order_status BOOLEAN,
+    PRIMARY KEY (order_id) NOT ENFORCED
+) WITH (
+    'connector' = 'oceanbase-cdc',
+    'scan.startup.mode' = 'initial',
+    'username' = 'user@test_tenant',
+    'password' = 'pswd',
+    'tenant-name' = 'test_tenant',
+    'database-name' = 'test_db',
+    'table-name' = 'orders',
+    'hostname' = '127.0.0.1',
+    'port' = '2881',
+    'rootserver-list' = '127.0.0.1:2882:2881',
+    'logproxy.host' = '127.0.0.1',
+    'logproxy.port' = '2983');
+
+-- 从表 orders 中读取快照数据和 binlogs
+Flink SQL> SELECT * FROM orders;
+```
+
+您也可以访问 Flink 官网文档，快速体验将数据从 OceanBase 导入到 Elasticsearch。更多信息，参考 [Flink 官网文档](https://ververica.github.io/flink-cdc-connectors/release-2.2/content/%E5%BF%AB%E9%80%9F%E4%B8%8A%E6%89%8B/oceanbase-tutorial-zh.html)。
+
+## Flink OceanBase CDC 连接器选项
+
```

Comment:
A large paragraph is missing before this table compared to the English version.

Reply:
Your source?
I'm using https://github.com/ververica/flink-cdc-connectors/blob/release-2.2/docs/content/connectors/oceanbase-cdc.md